tools/service-account-provider/README.md

```diff
@@ -0,0 +1,35 @@
+# Service Account Provider (SAPRO)
```

Comment:
The Readme is missing instructions on how to setup and use the tool

Reply:
I added some instructions. Thanks for pointing this out.